config/crd/bases/infrastructure.cluster.x-k8s.io_cloudstackmachinetemplates.yaml

```diff
@@ -103,6 +103,11 @@ spec:
                       diskOffering:
                         description: CloudStack disk offering to use.
                         properties:
+                          customSizeInGB:
+                            description: disk size if disk offering is custom disk
```

Comment:
I think we should align on whether we do full sentences with punctuation in descriptions or not.

Just above this description we did use a proper sentence and here we did not. I think we should use full sentences, but I'm mostly concerned with consistency.